src/x509.c

```diff
@@ -687,16 +687,9 @@ WOLFSSL_X509_EXTENSION* wolfSSL_X509_set_ext(WOLFSSL_X509* x509, int loc)
         /* Check if extension has been set */
         isSet = wolfSSL_X509_ext_isSet_by_NID((WOLFSSL_X509*)x509, nid);
         ext->obj = wolfSSL_OBJ_nid2obj(nid);
-        if (ext->obj == NULL) {
-            WOLFSSL_MSG("\tfail: Invalid OBJECT");
-            wolfSSL_X509_EXTENSION_free(ext);
-            FreeDecodedCert(cert);
-        #ifdef WOLFSSL_SMALL_STACK
-            XFREE(cert, NULL, DYNAMIC_TYPE_DCERT);
-        #endif
-            return NULL;
+        if (ext->obj != NULL) {
```

Comment:
Continuing on without a break if ext->obj is null results in potential null derefrences. An example would be if wolfSSL_OBJ_nid2obj failed and returned null and the case BASIC_CA_OID is used.

Reply:
Woah!!! Sharp eye @JacobBarthelmeh !!  Thank you very much for spotting that. Its been fixed.